exercises/parallel-letter-frequency/example.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from collections import Counter
+from threading import Lock, Thread
+from time import sleep
+import sys
+if sys.version[0] == '2':
+    from Queue import Queue
+else:
+    from queue import Queue
+
+total_workers = 3  # Maximum number of threads chosen arbitrarily
+
+
+class LetterCounter(object):
+
+    def __init__(self):
+        self.lock = Lock()
+        self.value = Counter()
+
+    def add_counter(self, counter_to_add):
+        self.lock.acquire()
+        try:
+            self.value = self.value + counter_to_add
+        finally:
+            self.lock.release()
+
+
+def count_letters(queue_of_texts, letter_to_frequency, worker_id):
+    while True:
```

Comment:
Generally it's a bad practice to you use infinite loops, but I think you know it
Maybe something like this:
```python
while not queue_of_texts.task_done():
    # do stuff
```

Reply:
Thank you for the feedback. I removed this in my latest commit.